pkg/arvo/lib/metadata-json.hoon

```diff
@@ -0,0 +1,64 @@
+/-  *metadata-store
+|%
+++  associations-to-json
+  |=  =associations
+  =,  enjs:format
+  ^-  json
+  %+  frond  %metadata-initial
+  %-  pairs
+  %+  turn  ~(tap by associations)
+  |=  [[=group-path =resource] =metadata]
+  ^-  [cord json]
+  :_  (metadata-to-json metadata)
+  %-  crip
+  ;:  weld
+      (trip (spat group-path))
+      (trip app-name.resource)
+      (trip (spat app-path.resource))
+  ==
+::
+++  metadata-to-json
+  |=  met=metadata
+  ^-  json
+  =,  enjs:format
+  %-  pairs
+  :~  [%title s+title.met]
+      [%description s+description.met]
+      [%color s+(scot %ux color.met)]
+      [%date-created s+(scot %da date-created.met)]
+      [%creator s+(scot %p creator.met)]
```

Comment:
Probably don't want to bother the frontend with urbit-style data notation. We have dedicated json transformers for `@da` and `@p`.

```hoon
> ((x-co:co 6) 0xff)
"0000ff"
```

https://github.com/urbit/urbit/blob/686aaa565eddfc0311d2e0f43564d4355f4173ec/pkg/arvo/sys/zuse.hoon#L5590-L5592

https://github.com/urbit/urbit/blob/686aaa565eddfc0311d2e0f43564d4355f4173ec/pkg/arvo/sys/zuse.hoon#L5575-L5577

Reply:
This is a concern for later. The frontend is used to dealing with renders that look like these, and I don't want to change them all wholesale right now.

Reply:
Dang, I should've caught this earlier then. But both this and the empty title thing signal that we're pushing a lot of things out to the frontend. Questionable imo, but a later discussion.